### DISCUSSION TITLE
Existing MET build(s) on NCAR HPC Derecho?

### DISCUSSION THREAD

**Discussion by michelleharrold** [dtcenter/METplus#2462] [Existing Builds] [answered]
Hi there!

I wanted to use MET on NCAR's HPC Derecho. I checked the [Existing Builds](https://dtcenter.org/community-code/metplus/metplus-5-1-existing-builds) webpage, but I could not find it. Are there any MET builds on Derecho? If not, are there plans to add an existing build to Derecho?

Thanks!
Michelle Harrold

**Comment by jprestop** [comment, marked as answer]
Hi @michelleharrold.  Thanks for your question.  I had installed MET-11.1.0 on Derecho, but unfortunately it was not in the correct area.  Just before I left for break, I received a new location to install MET and a role account to use.  It is on my list to install MET-11.1.0 very soon (perhaps today), and I will update here and on the Existing Builds page once that work is complete.  Will MET-11.1.0 work for you or would you like me to install a different version in addition to MET-11.1.0?

**Reply by michelleharrold** [reply]
@jprestop -- MET-11.1.0 would be perfect! (But no huge rush!). Thanks a ton!

**Reply by jprestop** [reply]
Thanks @michelleharrold!

**Comment by jprestop** [comment]
Hi @michelleharrold .  I have re-installed MET and METplus on both Casper and Derecho.  You can find the appropriate modules to load on our METplus [Existing Builds](https://dtcenter.org/community-code/metplus/metplus-5-1-existing-builds) page under "NCAR machines" for the 5.1 coordinated release.  I hope to have METcalcpy, METdataio, and METplotpy installed there as well by early next week.  Please let us know if you have any questions or problems.  I will install MET-11.1.1 once it's available.

**Reply by michelleharrold** [reply]
@jprestop -- thank you so much!
